### PR TITLE
MkDocs

### PR DESCRIPTION
**Pull request by fao89**
PoC of using MkDocs to publish the docs
https://fao89.github.io/data-plane-adoption/

### CODE REVIEW

**Comment by fao89** [comment]
@jistr @fmount 
I started this MkDocs PoC, feel free to take or close it, as I'll be out next week
- https://github.com/fao89/data-plane-adoption/tree/mkdocs
- https://fao89.github.io/data-plane-adoption/

**Comment by jistr** [comment]
Chatted with Fabricio, we should amend the left sidebar navigation to be multi-level, something like:

- OpenStack
  - Backend
  - MariaDB
  - Keystone
  - Glance
  - etc.
- Ceph
  - RBD
  
  Other than that the rendering looks good to me. I'll test the local site generation too and then i'm +1.

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - README.md
+  - OpenStack:
+    - backend_services_deployment.md
+    - mariadb_copy.md
+    - keystone_adoption.md
+    - glance_adoption.md
+    - other_services_adoption.md
+  - Ceph:
+    - ceph.md
+  - troubleshooting.md
```
Comment (fao89) [inline mkdocs.yml:37]: @jistr FYI this is how we define the sidebar

**Comment by jistr** [comment]
Looks good, and local docs building WFM. :+1: Let's discuss with others on the call today.

File: mkdocs.yml (hunk 1 above)
Comment (fmount) [reply mkdocs.yml:37]: This is actually nice and really useful, I'm going to have the RGW adoption section under `Ceph`, so it's nice I can define I list of documents to provide a better navigation!

**Comment by fultonj** [comment]
IMO we should merge this. If more people want to write docs in a different format they can do so, but we can't wait for the perfect time. Some docs are better than no docs.

**Comment by fmount** [comment]
> IMO we should merge this. If more people want to write docs in a different format they can do so, but we can't wait for the perfect time. Some docs are better than no docs.

Agree w/ John here, and given great part of the value of this repo is the doc/procedures that contains, +1 on merging it!

**Comment by fao89** [comment]
I'm gonna merge it then

**Comment by jistr** [comment]
:+1: Thanks Fabricio.

**Pull request merged**